LRThreeDigits.cpp

```diff
@@ -157,25 +164,22 @@ volatile uint16_t gDigitOutputMask[cDigitCount] = {
 
 /// Get the port D mask part from the combined mask.
 ///
-/// This `constexpr` will allow the compiler to resolve this function call
-/// into the optimized code.
-///
 /// @param mask The combined mask.
 /// @return The mask for port D.
 ///
-constexpr uint8_t getPortDMask(const uint16_t mask)
```

Comment:
The `constexpr` was in place, so the compiler will actually eliminate this method and just access the lower or higher part of the word, which leads to very efficient code in the interrupt method `updatePorts()`. It was meant as kind of helper method, not for extension.